packages/channels/botpress-channel-web/src/views/web/convo/index.jsx

```diff
@@ -161,7 +164,7 @@ export default class Convo extends React.Component {
             text={this.props.text}
             config={this.props.config}
           />
-          <div className={style.bottom}>
+          <div className={`${style.bottom} botpress__bottom`}>
```

Comment:
Again, this is _not_ the bottom of the entire webchat, right? It's like `botpress__new-message-footer`